src/main/java/com/epam/ta/reportportal/core/user/impl/DeleteUserHandler.java

```diff
@@ -79,6 +84,8 @@ public OperationCompletionRS deleteUser(String userId, String principal) {
 			throw new ReportPortalException("Error while deleting user", exp);
 		}
 
+		logIndexer.deleteIndex(user.getLogin().toLowerCase() + PersonalProjectService.PERSONAL_PROJECT_POSTFIX.toLowerCase());
```

Comment:
this is not right way to get personal project name. Use ProjectRepository.findPersonalProjectName